src/iaphandler.cpp

```diff
@@ -164,6 +164,17 @@ void IAPHandler::startSubscription(const QString& productIdentifier) {
   nativeStartSubscription(product);
 }
 
+void IAPHandler::startRestoreSubscription() {
+  logger.debug() << "Starting the restore of the subscription";
+
+#ifdef MVPN_IOS
+  nativeRestoreSubscription();
+#else
```

Comment:
You can implement the native method in the android component and log the error there. What do you think?